l10n_ch_invoice_reports/README.rst

```diff
@@ -83,6 +83,6 @@ OCA, or the Odoo Community Association, is a nonprofit organization whose
 mission is to support the collaborative development of Odoo features and
 promote its widespread use.
 
-This module is part of the `OCA/l10n-switzerland <https://github.com/OCA/l10n-switzerland/tree/15.0/l10n_ch_invoice_reports>`_ project on GitHub.
+This module is part of the `OCA/sl10n-switzerland <https://github.com/OCA/sl10n-switzerland/tree/16.0/l10n_ch_invoice_reports>`_ project on GitHub.
```

Comment:
thx, but why `l10n` was updated to `sl10n`?

Reply:
Thank for notice, this typo error from my side, when using wrong repo name generate readme file
Branch updated